### PR TITLE
Change navbar links and logo position

### PR DESCRIPTION
**Pull request by RoPy73**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -81,11 +81,11 @@
   
           <div class="slides-description-list">
             <div class="slide-description">Universe, 2016</div>
-            <!-- <div class="slide-description">Nikola Tesla Tower, 2020</div>
```
Comment (NikitaPytalev) [inline index.html:84]: This does not belong here, but we can leave it as is for now.

File: pytalev.html [hunk 2]
```diff
@@ -113,20 +113,9 @@ <h1>Who is Pytalev?</h1>
       function showButton() {
         window.scrollTo(pageXOffset, 0);
       };
-
       window.addEventListener('scroll', function() {
         var pytalevPageButton = document.querySelector('#pytalev-page-button');
         pytalevPageButton.hidden = (pageYOffset < document.documentElement.clientHeight/2);
-
```
Comment (NikitaPytalev) [inline pytalev.html:120]: Same here

File: styles.css [hunk 3]
```diff
@@ -334,7 +334,7 @@ html {
 
 .navbar-logo {
     float: left;
-    margin: 1rem;
+    margin: 1rem 5rem;
```
Comment (NikitaPytalev) [inline styles.css:337]: Looks good. Good job!

File: index.html (hunk 1 above)
Comment (RoPy73) [reply index.html:84]: Ups...

File: pytalev.html (hunk 2 above)
Comment (RoPy73) [reply pytalev.html:120]: Ups too...

File: styles.css (hunk 3 above)
Comment (RoPy73) [reply styles.css:337]: Thanks a lot

**Pull request merged**